### ISSUE TITLE
powiatslubicki.pl

### ISSUE THREAD

**Issue by krystian3w** [NanoAdblocker/NanoFilters#93]
### URLs where this issue occurs (Required)

```
https://powiatslubicki.pl/
```

### Screenshots (Required)

![image](https://user-images.githubusercontent.com/35370833/40915064-e99f1146-67fa-11e8-8b69-93a807646756.png) |
--- |

### Screenshot of console (Required, press `F12` to open the console)

N/A

### Describe the issue (Optional if obvious)

GDPR on main page. 

Could possible create reddirect to "Strona główna" (e.g simulate click link "Strona główna" when open main page `https://powiatslubicki.pl/`)?

`https://greasyfork.org/pl/scripts/369153-supplement-for-filterlist-polish-gdpr-cookies-filters-supplement-for-ublock-adguard/code` - workaround with  Violentmonkey/Tampermonkey/Greasemonkey

https://github.com/MajkiIT/polish-ads-filter/issues/7868

### Reproduction Steps (Optional if trivial)

1. open main website `https://powiatslubicki.pl/`
2. you should see RODO (GDPR) rules

#### Environment (Required)

- Operating System and Version:  **Windows 7 Pro**
- Browser and Version:  **Chrome 67.0.3396.62**
- Nano Adblocker Version:  **1.0.0.52**
- Nano Defender Version (Optional if you are not using it):  **N/A**

#### Your filter lists (Required)

Default Nano Adblocker + `POL: Official Polish filters for AdBlock, uBlock Origin & AdGuard`

#### Your custom filters (Required if you have any)

N/A

**Comment by jspenguin2017** [comment]
We don't have URL rewrite for now.

**Comment by hawkeye116477** [comment]
@jspenguin2017 Accept button is not available. There is only button to „real" main page here ("Strona główna"). 
I guess that, this rule should probably help `powiatslubicki.pl##+js(nanop-click-elements-onready.js, a[href="/strona-glowna"])` :smile:.

**Comment by krystian3w** [comment]
I'm afraid of the bootlop.

**Comment by hawkeye116477** [comment]
Yeah, right, that rule should only work on `https://powiatslubicki.pl/` and not on subpages just like on my userscript.

**Comment by jspenguin2017** [comment]
I can't find `Strona główna`, can you show it in a screenshot?

**Comment by jspenguin2017** [comment]
Oh you mean `Homepage` button? Eh this will be hard with existing script snippets...

**Comment by jspenguin2017** [comment]
I need a logic that does "if the text of this is that, click the other button", this is extremely complex unless I create a special script snippet for it, which is hardly worth for just annoyance. 

I'm thinking of creating a lightweight bytecode virtual machine because I'm not comfortable with pushing arbitrary JavaScript from filter lists, even for privileged filters. 

A system like that would also help with https://github.com/NanoAdblocker/NanoFilters/issues/91

**Comment by jspenguin2017** [comment]
Is the maintainer of `I don't care about cookies` on GitHub?

**Comment by krystian3w** [comment]
Have account - @kiboke

**Comment by kiboke** [comment]
I know how to do it with javascript, if that's what you're looking for, it's not complex at all. Btw. these rules are visible also here powiatslubicki.pl/de/ and here powiatslubicki.pl/en/

**Comment by jspenguin2017** [comment]
How would you do it? Test `location.href` with `/^https:\/\/powiatslubicki\.pl\/\w+\/$/` and redirect to home page?

**Comment by jspenguin2017** [comment]
Yea, I see what you mean. And it's kind of tricky to implement a fix.

**Comment by hawkeye116477** [comment]
It can be closed, cuz they „repaired" it (now is modal).
